src/main/kotlin/uk/gov/justice/digital/hmpps/approvedpremisesapi/service/cas2/StatusUpdateService.kt

```diff
@@ -35,6 +39,8 @@ class StatusUpdateService(
     return findActiveStatusByName(statusUpdate.newStatus) != null
   }
 
+  // too many returns, but unsure how to refactor to reduce
```

Comment:
i'm ok with not having the comment here as the SuppressWarnings makes it clear it was considered